kubernetes/cray-istio-operator/charts/istio-operator/templates/clusterrole.yaml

```diff
@@ -57,9 +57,7 @@ rules:
   - daemonsets
   - deployments
   - deployments/finalizers
-  - ingresses
   - replicasets
-  - statefulsets
```

Comment:
Why are these being removed? Does this have an impact on istio being able to inject the istio-proxy pods into statefulsets and ingresses?

Reply:
The operator chart was extracted from upstream, so I guess the 2 were not needed for istio-operator. In my testing, I didn't see an impact of injecting istio-proxy pods into statefulsets and ingresses. I'll keep an eye on it on other testing results if it turns out to be an issue.